setup.cfg

```diff
@@ -1,17 +1,23 @@
-[build_sphinx]
-source-dir = docs/source
-build-dir  = docs
-all_files  = 1
-
-[upload_sphinx]
-upload-dir = docs/html
```

Comment:
We may need to keep the sphinx configuration. :-/ It seems used by the Plone 5 documentation:
https://docs.plone.org/external/plone.app.caching/docs/index.html

Is there any way to keep this with current meta? We could put it back by hand for now.

Reply:
I re-added it manually for now.